src/app/GitUI/GitUI.csproj

```diff
@@ -9,6 +9,10 @@
     <ForceDesignerDpiUnaware>true</ForceDesignerDpiUnaware>
      <!-- Suppress suggestion to use experimental API ShowDialogAsync -->
     <NoWarn>$(NoWarn);WFO5002</NoWarn>
+
+    <!-- Enable C#13 keyword "field" -->
+    <LangVersion>preview</LangVersion>
```

Comment:
Why not set it at the solution level?

Reply:
and it should be 13.0 now as .net9 is the default